service-search-nls/src/main/java/fi/nls/oskari/search/channel/ELFGeoLocatorSearchChannel.java

```diff
@@ -448,12 +448,14 @@ public void calculateCommonFields(final SearchResultItem item) {
         try {
             JSONArray fuzzyHits = jsonObject.getJSONArray("fuzzy_search").getJSONObject(0).getJSONArray("options");
             for (int i = 0; i < fuzzyHits.length(); i++) {
-                combiner.addHit(fuzzyHits.getJSONObject(i), 0);
+                combiner.addHit(fuzzyHits.getJSONObject(i), false);
             }
 
-            JSONArray exactHits = jsonObject.getJSONArray("normal_search").getJSONObject(0).getJSONArray("options");
-            for (int i = 0; i < exactHits.length(); i++) {
-                combiner.addHit(exactHits.getJSONObject(i), 1000);
+            JSONArray normalHits = jsonObject.getJSONArray("normal_search").getJSONObject(0).getJSONArray("options");
+            for (int i = 0; i < normalHits.length(); i++) {
+                JSONObject hit = normalHits.getJSONObject(i);
+                boolean isExact = searchString.trim().equalsIgnoreCase(hit.getString("text"));
```

Comment:
Could move the trim() outside of the loop, however that might get optimized by the compiler anyway.

Reply:
Yes, youre right, but I left this as is, beacause the performance impact is minimal with the amount of results we have here. I think it's more readable this way.